scripts/mint.py

```diff
@@ -9,24 +8,40 @@
 from .utils import *
 
 
-def main(deployer=None, skip_confirmation=False):
-    if deployer is None:
-        deployer = accounts[0]  # for dev environment
+def main(execute_tx, deployer_account=None, priority_fee='2 wei', max_fee='300 gwei', skip_confirmation=False):
+    """`deployer_account` in live environment can either be:
+        - brownie account name: use the account
+        - None: save tx calldata for use in multisig
+    """
+    if not skip_confirmation:
+        assert not get_is_live()
+    
+    if type(execute_tx) != bool:
+        assert execute_tx in ['True', 'False'], "'execute_tx' can only by True or False"
+        execute_tx = True if execute_tx == 'True' else False
     
-    print(f'DEPLOYER is {deployer}')
+    if execute_tx:
```

Comment:
it's not clear from the caller's perspective, whether we use 'dry-run' mode or actually execute tx. Maybe add some note or confirmation?

Reply:
expanded this

    reply = input(f'The transaction will be executed on chain ({network.show_active()}).\n'
                           f'Are these parameters correct? (yes/no)\n')
                          
good?

Reply:
yep!